src/components/pickers/Suggestions/Suggestions.scss

```diff
@@ -66,6 +66,6 @@
   .ms-Spinner-label {
     display: inline-block;
     margin: 0px 10px;
-    vertical-align: middle;
+    vertical-align: top;
```

Comment:
is this right? normally i would expect some line height here I think. We do want it centered vertically I think.